src/transformers/generation/utils.py

```diff
@@ -1156,6 +1156,7 @@ def generate(
         prefix_allowed_tokens_fn: Optional[Callable[[int, torch.Tensor], List[int]]] = None,
         synced_gpus: Optional[bool] = None,
         assistant_model: Optional["PreTrainedModel"] = None,
+        low_memory: Optional[bool] = False,
```

Comment:
Sorry, I've missed this -- this should be part of the `GenerationConfig` class, and not an argument to `.generate()`. We want to avoid adding arguments directly on `.generate` :)

Reply:
Gotcha, I will move the kwarg over to the config.